pandas/tests/groupby/test_function.py

```diff
@@ -1398,6 +1398,38 @@ def test_quantile_array_multiple_levels():
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("frame_size", [(2, 2), (100, 10)])
+@pytest.mark.parametrize("groupby", [[0], [0, 1]])
+@pytest.mark.parametrize("q", [[0.5, 0.6], [0.1, 0.9]])
+def test_groupby_quantile_with_arraylike_q_and_int_columns(frame_size, groupby, q):
+    # GH30289
+    nrow, ncol = frame_size
+    if len(groupby) >= ncol or any(by >= ncol for by in groupby):
+        pytest.skip(f"Invalid argument groupby={groupby}")
```

Comment:
hmm, what case hits this? can we just make the cases so this doesn't happen

Reply:
it hits the skip when the groupby columns are not found or the test used all columns to groupby and nothing left for content (i.e when frame_size=(2, 2) and groupby=[0, 1]). Have cleaned up the test cases a bit so the skip is no longer there